daisy_workflows/image_import/import_image.sh

```diff
@@ -48,22 +53,48 @@ function resizeDisk() {
   fi
 }
 
-# Mount GCS bucket containing the disk image.
-mkdir -p /gcs/${SOURCEBUCKET}
-gcsfuse --implicit-dirs ${SOURCEBUCKET} /gcs/${SOURCEBUCKET}
+function copyImageToScratchDisk() {
+  local sourceSizeGigabytes="$((1 + SOURCE_SIZE_BYTES / 1024 / 1024 / 1024))"
```

Comment:
ref to export_disk_ext.sh about how to calculate size_gb. I think this line always grant one extra GB if source disk is a size of exact X GB.

Reply:
done!